docs/content/testnet/trisa-cli.en.md

```diff
@@ -24,7 +24,7 @@ Before you can start using the TRISA CLI, you must first configure your environm
 **Prerequisites**:
 
 1. The `trisa` command installed and on your `$PATH`
-2. Your [testnet certificates]({{< ref "/gds/registration" >}}) that include both the trust chain and private key.
+2. Your [testnet certificates]({{< ref "/joining-trisa/registration" >}}) that include both the trust chain and private key.
```

Comment:
did not change in [da05ec2](https://github.com/trisacrypto/trisa/pull/101/commits/da05ec2b976a211552b07f3afbb47e3582919d7c) because: 
- for  `faq.en.md`, the` /gds` refers to working with GDS and not registration
- for `secure-envelopes/_index.en.md ` it seems to refer directly to the GDS docs as well